exec/java-exec/src/main/java/org/apache/drill/exec/expr/stat/ParquetBooleanPredicate.java

```diff
@@ -40,51 +40,72 @@ private ParquetBooleanPredicate(String name, List<LogicalExpression> args, Expre
   }
 
   @SuppressWarnings("unchecked")
-  private static <C extends Comparable<C>> LogicalExpression createAndPredicate(
-      String name,
-      List<LogicalExpression> args,
-      ExpressionPosition pos
-  ) {
+  private static <C extends Comparable<C>> LogicalExpression createAndPredicate(String name, List<LogicalExpression> args, ExpressionPosition pos) {
```

Comment:
Please keep the original format. Note that the new format makes it harder to review on the github.

Reply:
Done